lib/file-pipeline/read.js

```diff
@@ -24,7 +24,7 @@ export function read(context, file, next) {
   if (file.value || file.data.unifiedEngineStreamIn) {
     debug('Not reading file `%s` with `value`', filePath)
     next()
-  } else if (statistics(file).fatal) {
+  } else if (statistics(file).fatal || file.data.unifiedEngineIgnored) {
     debug('Not reading failed file `%s`', filePath)
```

Comment:
I understand the control flow is the same, but the debug logging no longer represents the situation: an ignored file is not a failed file.

```suggestion
    debug('Not reading failed or ignored file `%s`', filePath)
```

```suggestion
    if (file.data.unifiedEngineIgnored) {
      debug('Not reading ignored file `%s`', filePath)
    } else {
      debug('Not reading failed file `%s`', filePath)
    }
```

I feel like my latter suggestion is a bit verbose for correcting a debug message most people don’t even see though.